tests/functional/metadata/mongodb/listObject.spec.js

```diff
@@ -83,6 +113,7 @@ describe('MongoClientInterface::metadata.listObject', () => {
                         readPreference: 'primary',
                         database: DB_NAME,
                     },
+                    customListingParser,
```

Comment:
we don't want to use the custom parser always : otherwise, we do not cover the standard/default parser anymore... (which can be seen in the codecov report: https://app.codecov.io/github/scality/Arsenal/commit/52875fea66ecf04a76261592a4e825b833ce8858)

Also, (except for the "location" part), these tests validate that indeed the default parser does the proper mapping (which do not make sense to test with your change: since now we test that the custom parser does the proper mapping).

→ it should be more appropriate to create a separate test case, with a very simple custom parser (e.g. return parsed json object, no mapping!) : since listObject seems quite agnostic to the actual content of the parsed objects...
→ if that does not work, another approach may be to write a custom parser which "wraps" the default parser (`_parseListEntries`) after asserting the location field is not present. This could be used on all tests, but we would not verify that indeed this function is used as the default parser: hence it would be preferable to use the "separate test case" approach